content/content-tool/tool/src/webapp/vm/resources/sakai_properties.vm

```diff
@@ -31,6 +31,16 @@
             $('#access_groups${DOT}'+index.value).closest('.multiselect-native-select').replaceWith(content);
         });
     });
+
+    function copyToClipboard() {
+        var input = document.getElementById('fileURLHolder');
+        input.style.overflow='auto';
```

Comment:
put a null check in here and maybe make it a const? Maybe put spaces around the = ? :)

Reply:
sounds good

Reply:
good